### PR TITLE
clang-format380

### PR DESCRIPTION
**Pull request by Ashton-W**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-versions/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?
- [x] Does your submission adhere to Versions "Acceptable Formulae" [guidelines](https://github.com/Homebrew/homebrew-versions/blob/07b17dfb9f1/README.md#acceptable-formulae)?

-----

clang-format 3.8.0

### CODE REVIEW

File: clang-format380.rb [hunk 1]
```diff
@@ -0,0 +1,59 @@
+class ClangFormat380 < Formula
+  desc "Formatting tools for C, C++, ObjC, Java, JavaScript, TypeScript"
+  homepage "http://clang.llvm.org/docs/ClangFormat.html"
+  version "2016-03-08"
```
Comment (MikeMcQuaid) [inline clang-format380.rb]: Should be `3.8.0`

File: clang-format380.rb [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      sha256 "36804511b940bc8a7cefc7cb391a6b28f5e3f53f6372965642020db91174237b"
+    end
+  end
+
+  bottle do
+    cellar :any_skip_relocation
+    sha256 "e6b26890dc59982f2beb0bf45a6de51c5f63dbcc9739f583a84c47495eae2fa3" => :el_capitan
+    sha256 "f6cd9d90f3acbac3cfbce746ba476f3c1b8633787a9d6bddef1d12dae2f1b563" => :yosemite
+    sha256 "114b346b627ffb49e03d28fb20d0ebe6380090478795cc43b503ba841db4016c" => :mavericks
+  end
```
Comment (MikeMcQuaid) [inline clang-format380.rb]: Can remove this.

**Comment by MikeMcQuaid** [comment]
Go for `clang-format38` instead of `380`.

**Comment by Ashton-W** [comment]
Made the changes discussed. I added a comment to note the version as it was in homebrew-core, hope that is OK.

File: clang-format38.rb [hunk 3]
```diff
@@ -0,0 +1,53 @@
+class ClangFormat38 < Formula
+  desc "Formatting tools for C, C++, ObjC, Java, JavaScript, TypeScript"
+  homepage "http://clang.llvm.org/docs/ClangFormat.html"
+  # was version 2016-03-08 in homebrew-core
+  version "3.8.0"
+
+  conflicts_with "clang-format", :because => "Differing versions of the same formula"
```
Comment (DomT4) [inline clang-format38.rb]: This belongs under the `depends_on` statements. An empty line between 'em though.

File: clang-format38.rb [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+  stable do
```
Comment (DomT4) [inline clang-format38.rb]: Unless we're planning to add `head` here, you can drop the `stable do` block and nudge everything back two spaces.

**Comment by Ashton-W** [comment]
I can rebase before the merge if neccesary

**Comment by apjanke** [comment]
@BrewTestBot test this please

**Pull request closed without merging**